### PR TITLE
Preliminary Spanish translation

### PR DESCRIPTION
**Pull request by theotocopulitos**
## Status
**IN DEVELOPMENT**

## Migrations
YES | NO

## Description
Preliminary Spanish translation, form the docs. It has not yet been tested on the frontend [#332] 

Some parts will need revision once I see them in the actual interface... these have been marked as '******' for ease of finding.

Some texts might need readjusting since Spanish sentences are way longer than English equivalents.

### CODE REVIEW

**Comment by mcpierce** [comment]
@theotocopulitos A few initial suggestions.

1. Please update each commit message to, in this case, start with the word "Added" (since you're adding a new feature with the translation).
1. Add to the end of the first line in each commit the following: ```[#348]```
   * This will update the ticket with a reference to your commit so we can cross reference things

See [the contributing file](https://github.com/comixed/comixed/blob/develop/CONTRIBUTING.md) for what the commit messages need to look like.

File: comixed-frontend/src/assets/i18n/es/comics.json [hunk 1]
```diff
@@ -2,283 +2,283 @@
...
       },
       "error": {
-        "detail": "Failed to save page changes."
+        "detail": "Error al guardar los cambios en la página."
       }
     },
     "set-page-hash-blocking": {
       "success": {
-        "detail": "Successfully {state, select, true{blocked} other{unblocked}} pages with the hash {hash}."
+        "detail": "Páginas con hash {hash} {state, select, true{'bloqueadas'} other{'desbloqueadas'}} correctamente."
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/comics.json]: Please remove the single quotes around the translated values here: you only need the text within the curly braces.

File: comixed-frontend/src/assets/i18n/es/library.json [hunk 2]
```diff
@@ -2,484 +2,485 @@
...
-      "header": "Set Blocking State For Pages",
-      "message": "Are you sure you want to {blocking, select, true{block} other{unblock}} the selected pages?"
+      "header": "Establecer estado de 'bloqueo' para las páginas",
+      "message": "¿Seguro que quieres {blocking, select, true{'bloquear'} other{'desbloquear'}} las páginas selecciondas?"
     },
     "set-deleted": {
-      "header": "Mark Pages For Deletion",
-      "message": "Are you sure you want to {deleted, select, true{mark} other{unmark}} the selected pages?"
+      "header": "Marcar páginas para eliminación",
+      "message": "¿Seguro que quieres {deleted, select, true{marcar other{desmarcar}} las páginas seleccionadas?"
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/library.json]: This entry is missing a closing curly brace on the true entry. Should be:

```true{marcar} other{desmarcar}```

File: comixed-frontend/src/assets/i18n/es/comics.json (hunk 1 above)
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/comics.json]: Please remove the single quotes around bloqueadas and desbloqeadas.

File: comixed-frontend/src/assets/i18n/es/library.json [hunk 3]
```diff
@@ -2,484 +2,485 @@
...
     "get-all": {
       "error": {
-        "detail": "Failed to get all duplicate pages."
+        "detail": "Error al obtener todas las páginas duplicadas."
       }
     },
     "set-blocking": {
       "success": {
-        "detail": "Successfully marked selected pages as {blocked, select, true{blocked} other{unblocked}}."
+        "detail": "Páginas seleccionadas marcadas correctamente como {blocked, select, true{'bloqueadas'} other{'desbloqueadas'}}."
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/library.json]: Please remove the single quotes from around the words bloqueadas and desbloqueadas.

File: comixed-frontend/src/assets/i18n/es/library.json [hunk 4]
```diff
@@ -2,484 +2,485 @@
...
-      "blocking-options": "Blocking",
-      "deleting-options": "Deleting"
+      "blocking-options": "Bloqueando*****",
+      "deleting-options": "Eliminando*****"
     },
     "set-blocking": {
-      "header": "Set Blocking State For Pages",
-      "message": "Are you sure you want to {blocking, select, true{block} other{unblock}} the selected pages?"
+      "header": "Establecer estado de 'bloqueo' para las páginas",
+      "message": "¿Seguro que quieres {blocking, select, true{'bloquear'} other{'desbloquear'}} las páginas selecciondas?"
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/library.json]: Please remove the single quotes from around bloqueaar and desbloquear.

File: comixed-frontend/src/assets/i18n/es/library.json [hunk 5]
```diff
@@ -2,484 +2,485 @@
...
-    "tooltip": "This page is {blocked, select, true{blocked} other{not blocked}}, and {deletedCount, plural, =0{none are} =1{# is} other{# are}} marked for deletion.",
-    "caption": "Found {count, plural, =1{one copy} other{# copies}}."
+    "tooltip": "Esta página está {blocked, select, true{'bloqueada'} other{no 'bloqueada'}}, and {deletedCount, plural, =0{ninguna está marcada} =1{# está marcada} other{# están marcadas}} para eliminación.",
+    "caption": "Encontrada{count, plural, =1{una copia} other{s # copias}}."
   },
   "duplicate-page-list-item": {
-    "title": "Found {count, plural, =1{one copy} other{# copies}}.",
-    "subtitle": "This page is {blocked, select, true{blocked} other{not blocked}}, and {deletedCount, plural, =0{none are} =1{# is} other{# are}} marked for deletion."
+    "title": "Encontrada{count, plural, =1{una copia} other{s # copias}}.",
+    "subtitle": "Esta página está {blocked, select, true{'bloqueada'} other{no 'bloqueada'}}, and {deletedCount, plural, =0{ninguna está marcada} =1{# está marcada} other{# están marcadas}} para eliminación."
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/library.json]: Please remove the single quotes from around bloqueada in both spots here.

File: comixed-frontend/src/assets/i18n/es/library.json [hunk 6]
```diff
@@ -2,484 +2,485 @@
...
-      "header": "Mark Pages For Deletion",
-      "message": "Are you sure you want to {deleted, select, true{mark} other{unmark}} the selected pages?"
+      "header": "Marcar páginas para eliminación",
+      "message": "¿Seguro que quieres {deleted, select, true{marcar other{desmarcar}} las páginas seleccionadas?"
     }
   },
   "duplicate-page-grid-item": {
-    "tooltip": "This page is {blocked, select, true{blocked} other{not blocked}}, and {deletedCount, plural, =0{none are} =1{# is} other{# are}} marked for deletion.",
-    "caption": "Found {count, plural, =1{one copy} other{# copies}}."
+    "tooltip": "Esta página está {blocked, select, true{'bloqueada'} other{no 'bloqueada'}}, and {deletedCount, plural, =0{ninguna está marcada} =1{# está marcada} other{# están marcadas}} para eliminación.",
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/library.json]: Please remove the single quotes from around bloqueada in both spots here.

**Review by mcpierce — CHANGES_REQUESTED** [review]
@theotocopulitos This overall looks pretty good. A few more requests and then we can merge this:

1. Please put both changes into a single commit.
1. Have the first line of the commit start with "Added" since it's adding a new translation.
1. Have the first line of the comment end with ```[#348]``` to reference that translation ticket.
1. Add yourself to the CONTRIBUTORS.md file since you're now contributing to the project! :smile:

Take a look [the CONTRIBUTOR.md](https://github.com/comixed/comixed/blob/develop/CONTRIBUTING.md#commit-template) for an example of the commit message format to use.

File: comixed-frontend/src/assets/i18n/es/comics.json (hunk 1 above)
Comment (theotocopulitos) [reply comixed-frontend/src/assets/i18n/es/comics.json]: Solved [2c233768abc812a9e6eea1fcd0bb9a654784f8d7]

Comment (theotocopulitos) [reply comixed-frontend/src/assets/i18n/es/comics.json]: Hopefully solved with commit [abd9ff7]

File: comixed-frontend/src/assets/i18n/es/library.json (hunk 5 above)
Comment (theotocopulitos) [reply comixed-frontend/src/assets/i18n/es/library.json]: Hopefully solved with commit [abd9ff7]

File: comixed-frontend/src/assets/i18n/es/library.json (hunk 3 above)
Comment (theotocopulitos) [reply comixed-frontend/src/assets/i18n/es/library.json]: Hopefully solved with commit [abd9ff7]

**Comment by theotocopulitos** [comment]
I am not sure I was able to merge all commits into one...

**Comment by mcpierce** [comment]
> I am not sure I was able to merge all commits into one...

Yeah, it looks like you did a merge of develop back into your branch, which created a synthetic commit. Instead, please do the following and then re-push your change:

1. On your branch do the following: ```$ git rebase -i develop```
1. Delete anything in the list that's not your commit.

This will pull all of the changes on develop to your branch and then put your changes on top of it. Otherwise there will be merge issues when your branch is merged.

There are also some merge issues in the translation commit that you'll need to fix. I'll add comments on them now so you can refer to them when you clean them up after rebasing.

File: comixed-frontend/src/assets/i18n/es/comic-import.json [hunk 7]
```diff
@@ -2,96 +2,114 @@
...
+      "select": "Seleccionar",
+      "import": "Importar"
     },
     "tooltip": {
-      "delete-blocked-pages": "Blocked pages will be marked for deletion.",
-      "allow-blocked-pages": "Blocked pages will not be marked for deletion.",
-      "start-import": "Import the selected comics.",
-      "list-layout": "Show comics as a list.",
-      "grid-layout": "Show comics as a grid."
+<<<<<<< HEAD
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/comic-import.json]: Please fix these merge conflict errors.

File: comixed-frontend/src/assets/i18n/es/comic-import.json [hunk 8]
```diff
@@ -2,96 +2,114 @@
...
-      "file-size": "File Size",
-      "search-comic-names": "Search Comic Names"
+      "filename": "Nombre del archivo",
+      "file-size": "Tamaño",
+      "search-comic-names": "Buscar nombres de cómics*****"
     },
     "text": {
-      "blocked-pages-notice": "Blocked pages in those comics will be automatically marked as deleted...",
-      "search-by-series": "Search by series name..."
+<<<<<<< HEAD
```
Comment (mcpierce) [inline comixed-frontend/src/assets/i18n/es/comic-import.json]: Please fix these merge conflict errors.

**Review by mcpierce — CHANGES_REQUESTED** [review]
We're very close. Just a few pieces to cleanup after rebasing and this should be good to merge.

File: comixed-frontend/src/assets/i18n/es/comic-import.json (hunk 7 above)
Comment (theotocopulitos) [reply comixed-frontend/src/assets/i18n/es/comic-import.json]: Hopefully resolved with commit [5d20c3e62a4e1e05a92a33bfcf50929b33436b26]

File: comixed-frontend/src/assets/i18n/es/comic-import.json (hunk 8 above)
Comment (theotocopulitos) [reply comixed-frontend/src/assets/i18n/es/comic-import.json]: Hopefully resolved with commit [5d20c3e]

**Comment by mcpierce** [comment]
Closed this since it was manually merged.

**Pull request closed without merging**